events/filters.py

```diff
@@ -52,3 +54,7 @@ def filter_by_event_global_id(self, qs, name, value):
 
     def filter_by_occurrence_language(self, qs, name, value):
         return qs.filter(occurrence_language=value.lower())
+
+    def filter_by_project_global_id(self, qs, name, value):
```

Comment:
It's really cool that for `venue` and `event` filters we don't need this function :D